docker/mkdocs/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.7.4-slim-buster
+FROM python:3.13-slim-bookworm
```

Comment:
What about to use `python:3.13-alpine` ?

Reply:
I'll try it